assets/templates/partials/release/status-header.tmpl

```diff
@@ -1,4 +1,27 @@
 <div>
+  <div class="ons-u-pb-s">
+    {{ if .Description.Census2021 }}
```

Comment:
Won't this create an empty div when not a census release?

Reply:
yes that true and i have fix this.